Detectors/MUON/MCH/Simulation/src/GeometryTest.cxx

```diff
@@ -136,6 +136,21 @@ void createRegularGeometry()
   o2::mch::Detector(true).ConstructGeometry();
 }
 
+void addAlignableVolumesToRegularGeometry()
+{
+  // First create the geometry
+  createRegularGeometry();
+
+  // If not closed, we need to close it
+  if (!gGeoManager->IsClosed()) {
+    std::clog << "Geometry is not closed, will do it now\n";
```

Comment:
must confess I did not know about clog (first thought it was a typo ;-) ). Given it's the only instance (as far as I can tell) in the full AliceO2 repo, I wonder what's the advantage ?

Reply:
No idea. Just thought std::cerr was for error messages and std::clog for just log messages. Then what is recommended and for my own education why is std::cerr used here instead of LOG(...)?

Reply:
Well, that's probably a personal thing but my rationale is to use cout/cerr for things that are only in tests and LOG for the parts than (will) end up in production code, as I don't see the advantage of LOG for tests. And no, I've not checked that all MCH/MID code is actually following this ;-)